zipline/pipeline/pipeline.py

```diff
@@ -37,6 +39,9 @@ def __init__(self, columns=None, screen=None):
 
         if columns is None:
             columns = {}
+        for term in columns.values():
```

Comment:
Can we push this into a `validate_column` method?  (We're duplicating this logic in `add`.)